float64 delta_pan
float64 delta_tilt
bool focus_hand
